src/TSMapEditor/Initialization/MapLoader.cs

```diff
@@ -995,7 +995,7 @@ public static void ReadAITriggerTypes(IMap map, IniFile mapIni)
                         AddMapLoadError($"AITriggerType {kvp.Key} has a non-existent condition object \"{parts[5]}\"");
                     }
 
-                    aiTriggerType.ConditionObjectString = parts[5];
+                    aiTriggerType.ConditionObject = map.Rules.FindTechnoType(parts[5]);
```

Comment:
No need to look up the object again when it's already done 7 lines earlier :)

```suggestion
                    aiTriggerType.ConditionObject = conditionObject;
```